app/lib/client/client.dart

```diff
@@ -32,14 +32,18 @@ class SPHclient {
   String password = "";
   String schoolID = "";
   String schoolName = "";
-  Map<SPHAppEnum, LoadApp>? applets;
+
+  final Map<SPHAppEnum, LoadApp> applets = {};
+  late Map<String, dynamic> shouldFetchApplets;
```

Comment:
Bitte füge docstings hinzu in dem du erklärst, was das genau für variablen sind.

Kann man den Type nicht "dynamic" machen?

Reply:
Man könnte sie dynamic machen, aber dann kriegt man keine Code Suggestions und so, weil es sich von nirgendwo ableiten kann.

Reply:
Ich meine das dynamic weg machen. Es scheint eine sehr definierte variable zu sein?

Reply:
Achso
Flutter hatte anscheinend Probleme das dort bool statt dynamic steht. Guck es mir gerade genauer an.